.readthedocs.yaml

```diff
@@ -2,27 +2,23 @@
 
 version: 2
 
-# Build documentation in the docs/ directory with Sphinx
-sphinx:
-  builder: dirhtml
-  configuration: docs/conf.py
-  fail_on_warning: true
-
-formats:
-  - pdf
-  - htmlzip
```

Comment:
So we're loosing these with the update. That's another thing we could mention in the change note.
Alternatively, we could try reproducing the same commands that RTD injects to get these outputs.

Reply:
I guess it's fine to drop these. Marking the thread as resolved.